tests/freqtradebot/test_worker.py

```diff
@@ -50,6 +70,73 @@ def test_worker_stopped(mocker, default_conf, caplog) -> None:
     assert mock_throttle.call_count == 1
 
 
+@pytest.mark.parametrize(
+    "old_state, target_state, expected_state,startup_call, log_fragment",
+    [
+        (State.STOPPED, State.PAUSED, State.PAUSED, True, "Changing state from STOPPED to: PAUSED"),
+        (
+            State.RUNNING,
+            State.PAUSED,
+            State.PAUSED,
+            False,
+            "Changing state from RUNNING to: PAUSED",
+        ),
+        (
+            State.PAUSED,
+            State.RUNNING,
+            State.RUNNING,
+            False,
+            "Changing state from PAUSED to: RUNNING",
+        ),
+        (
+            State.PAUSED,
+            State.STOPPED,
+            State.STOPPED,
+            False,
+            "Changing state from PAUSED to: STOPPED",
+        ),
+        (
+            State.RELOAD_CONFIG,
+            State.RUNNING,
+            State.RUNNING,
+            True,
+            "Changing state from RELOAD_CONFIG to: RUNNING",
+        ),
```

Comment:
I've modified this test slightly so it shows if "startup" was called or not.

i think this test (paused -> running) shows an edge-case - would we expect "startup" to be called here?